extra/changelog-generator/changelog-generator

```diff
@@ -167,9 +241,18 @@ if args.all:
         .decode()
         .strip()
     )
-    repos += [os.path.join(base_dir, repo) for repo in repo_list.split()]
+    repos += repo_list.split()
 else:
-    repos.append(".")
+    repos.append(os.path.basename(os.path.realpath(".")))
+
+if not args.base_dir and not args.repo and not args.query_github:
+    print("Need at least one of --base-dir, --repo or --query-github")
+    sys.exit(1)
+
+if args.query_github:
```

Comment:
Idea: 

Could we drop the `query_github` check, and do:

` locally` -> `!found` -> `github`

? :thinking: